src/containers/table-container-row/table-container-row.js

```diff
@@ -38,9 +38,9 @@ const TableContainerRow = ({
   const properties = { ...rest };
   const tableCells = Object.entries(properties).map(
     ([key, value]) =>
-      value && (
+      (value || value === 0) && (
```

Comment:
What types of value can you receive here?

Reply:
string or number, and if it is 0 it should show it

Reply:
U cant turn off the cell element on table

Reply:
fixed